file/reader_test.go

```diff
@@ -40,9 +40,9 @@ func Test_ensureJSON(t *testing.T) {
 }
 
 func TestReadKongStateFromStdinFailsToParseText(t *testing.T) {
-	var filename = "-"
+	var filenames = []string{"-"}
 	assert := assert.New(t)
-	assert.Equal("-", filename)
+	assert.Equal("-", filenames[0])
```

Comment:
There is an interesting case where some configuration can be specified via Stdin and other via flag.
I was considering to restrict `-` as standalone, meaning you either specify only file(s) or only Stdin.
I think it is okay to allow it.
Just leaving a thought. No changes required.